lib/base_jobmaster.go

```diff
@@ -32,6 +32,11 @@ type BaseJobMaster interface {
 	ID() worker.RunnableID
 	UpdateJobStatus(ctx context.Context, status WorkerStatus) error
 
+	// IsMasterReady returns whether the master has received heartbeats for all
+	// workers after a fail-over. If this is the first time the JobMaster started up,
+	// the return value is always true.
+	IsMasterReady() bool
```

Comment:
why implement this method in jobmaster instead of master

Reply:
It is implemented in master too. But we need to redefine it in JobMaster too.